### PR TITLE
feat(governance): updating picasso open gov

### PR DESCRIPTION
**Pull request by kkast**
Make PICA governance token on picasso:
1) remove opengovbalances token and clean pallet date
2) remove democracy  and clean pallet date. release locks with id "democrac" for pica
3) added call filter to disallow blocking of referenda, whitelist, convictionVoting pallets
4) added new tracks: GeneralAdmin, ReferendaCanceller, ReferendaKiller
5) changed rights for pallets to remove council and tech committee from making decisions
6) make council only responsible for treasury decisions 
7) moved relayer to the new RelayerCommittee
8) fixed tech committee instance gererics for picasso and composable
9) remove treasury from call filter

Required for merge:
- [ ] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](../terraform/github.com/branches.tf) 

Makes review faster:
- [ ] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [ ] I marked PR by `misc` label if it should not be in release notes
- [ ] Linked Zenhub/Github/Slack/etc reference if one exists
- [ ] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [ ] Added reviewer into `Reviewers`
- [ ] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [ ] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [ ] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

File: code/parachain/runtime/picasso/src/gates.rs [hunk 1]
```diff
@@ -85,8 +84,8 @@ impl Contains<RuntimeCall> for BaseCallFilter {
 
 impl call_filter::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type EnableOrigin = EnsureRootOrHalfNativeTechnical;
-	type DisableOrigin = EnsureRootOrOneThirdNativeTechnical;
+	type EnableOrigin = EnsureRoot<Self::AccountId>;
+	type DisableOrigin = EnsureRoot<Self::AccountId>;
```
Comment (dzmitry-lahoda) [inline code/parachain/runtime/picasso/src/gates.rs:88]: this must be uber fast. 

it must not allow to block sudo and referenda calls so I guess

Comment (kkast) [reply code/parachain/runtime/picasso/src/gates.rs:88]: if we do whitlist, does it count as uber fast. i can keep technical commitee. for not allowing to block, i can add a hook

File: code/parachain/runtime/picasso/src/governance.rs [hunk 2]
```diff
@@ -2,34 +2,33 @@
...
 }
 
 impl membership::Config<NativeCouncilMembership> for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type AddOrigin = EnsureRootOrTwoThirdNativeCouncil;
-	type RemoveOrigin = EnsureRootOrTwoThirdNativeCouncil;
-	type SwapOrigin = EnsureRootOrTwoThirdNativeCouncil;
-	type ResetOrigin = EnsureRootOrTwoThirdNativeCouncil;
-	type PrimeOrigin = EnsureRootOrTwoThirdNativeCouncil;
+	type AddOrigin = EnsureRoot<AccountId>;
```
[the remark this answers is not in the document]
Comment (kkast) [reply code/parachain/runtime/picasso/src/governance.rs:21]: ok, will return it back

[the remark this answers is not in the document]
Comment (kkast) [reply code/parachain/runtime/picasso/src/governance.rs:21]: bt this councilk becomes only for tresury decisions, but makes sense to make it govern itself

**Review by dzmitry-lahoda — DISMISSED** [review]

File: code/parachain/runtime/picasso/src/migrations.rs [hunk 3]
```diff
@@ -57,167 +69,55 @@ impl OnRuntimeUpgrade for TechCollectiveRenameMigration {
...
+		membership::pallet::Pallet::<Runtime, NativeRelayerMembership>::add_member(
+			frame_system::RawOrigin::Root.into(),
+			relayer_address.clone(),
+		)
+		.expect("should add");
+		membership::pallet::Pallet::<Runtime, NativeTechnicalMembership>::remove_member(
+			frame_system::RawOrigin::Root.into(),
+			relayer_address,
+		)
+		.expect("should remove");
```
Comment (vmarkushin) [inline code/parachain/runtime/picasso/src/migrations.rs:98]: Why do we need to add and remove?

File: code/parachain/runtime/picasso/src/tracks.rs [hunk 4]
```diff
@@ -22,36 +22,40 @@ const fn percent(x: i32) -> sp_runtime::FixedI64 {
 	sp_runtime::FixedI64::from_rational(x as u128, 100)
 }
 
+const fn permill(x: i32) -> sp_runtime::FixedI64 {
+	sp_runtime::FixedI64::from_rational(x as u128, 1000)
```
Comment (vmarkushin) [inline code/parachain/runtime/picasso/src/tracks.rs:26]: Why not to use the standard `Permill` type?

File: code/parachain/runtime/picasso/src/gates.rs [hunk 5]
```diff
@@ -76,18 +70,36 @@ pub struct BaseCallFilter;
...
+	#[inline(always)]
+	fn enable_hook(_: &CallFilterEntry<S>) -> DispatchResult {
+		Ok(())
+	}
+	#[inline(always)]
+	fn disable_hook(entry: &CallFilterEntry<S>) -> DispatchResult {
+		if entry.pallet_name.clone().into_inner() != b"Referenda".to_vec() &&
+			entry.pallet_name.clone().into_inner() != b"Sudo".to_vec() &&
+			entry.pallet_name.clone().into_inner() != b"Whitelist".to_vec() &&
+			entry.pallet_name.clone().into_inner() != b"ConvictionVoting".to_vec()
```
Comment (vmarkushin) [inline code/parachain/runtime/picasso/src/gates.rs:89]: `entry.pallet_name.clone().into_inner()` can be extracted to a variable

**Comment by dzmitry-lahoda** [comment]
let fix xcm first before merge

File: code/parachain/runtime/picasso/src/migrations.rs (hunk 3 above)
Comment (kkast) [reply code/parachain/runtime/picasso/src/migrations.rs:98]: i move hyperspace to a separate comittee. i do it in migration so hyperspace doesnt stop

File: code/parachain/runtime/picasso/src/tracks.rs (hunk 4 above)
Comment (kkast) [reply code/parachain/runtime/picasso/src/tracks.rs:26]: i dont think it suport FixedI64

**Review by dzmitry-lahoda — APPROVED** [review]

**Pull request merged**